### PR TITLE
Create Maze, Timer, and Move

### PR DESCRIPTION
**Pull request by Tomas-Ha**
Created Maze, Timer, and Move as well as added Maze to the BuilderSetup. Maze and Timer currently use direct System calls but that will be fixed in the future. @devan-srinivasan be sure to message me before closing this branch and make sure that my code follows the proper pattern.

### CODE REVIEW

File: .idea/misc.xml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
-  <component name="ProjectRootManager" version="2" languageLevel="JDK_11" default="true" project-jdk-name="corretto-11" project-jdk-type="JavaSDK">
+  <component name="ProjectRootManager" version="2" languageLevel="JDK_15" default="true" project-jdk-name="15" project-jdk-type="JavaSDK">
```
Comment (riyanah) [inline .idea/misc.xml:3]: I thought we're using java version 11

Comment (Tomas-Ha) [reply .idea/misc.xml:3]: I didn't think we did either, interesting

Comment (Tomas-Ha) [reply .idea/misc.xml:3]: > 

I didn't think we did either, interesting

Comment (Tomas-Ha) [reply .idea/misc.xml:3]: @devan-srinivasan Check this please

**Comment by riyanah** [comment]
Looks good

File: .idea/misc.xml (hunk 1 above)
Comment (devan-srinivasan) [reply .idea/misc.xml]: yea it should be `JDK 11`. Looks like you resolved it though great work!

**Pull request merged**